benchmark/bootstrap_panoptes.rb

```diff
@@ -33,45 +33,53 @@ def create_db_schema
     @pg.exec <<-SQL
```

Comment:
https://github.com/camallen/cellect_panoptes/blob/active_record/benchmark/bootstrap_panoptes.rb#L90

Instead of `classification_count`, it's `classifications_count`